hipscat/partitioner.py

```diff
@@ -475,21 +490,30 @@ def write_structure_metadata(self):
             print(dumped_metadata)
 
 
+    def construct_hips_structure(self):
+        assert self.opix is not None, 'bad'
+        self.hips_structure = self.opix
+
+
 if __name__ == '__main__':
     import time
     s = time.time()
     ###
     #urls = glob.glob('/epyc/data/gaia_edr3_csv/*.csv.gz')[:10]
 
-    urls = glob.glob('/epyc/data/sdss_parquet/*.parquet')[:10]
-    imp = Partitioner(catname='sdss', urls=urls, order_k=10, verbose=True, debug=True)
+    #client = Client(local_directory='/epyc/projects3/sam_hipscat/', n_workers=48, threads_per_worker=1)
+    urls = glob.glob('/epyc/data/gaia_edr3_csv/*.csv.gz')
```

Comment:
Let's not include stuff in library code that will only work in a specific environment.

Reply:
removed